app/models/katello/concerns/smart_proxy_extensions.rb

```diff
@@ -121,7 +133,16 @@ def set_default_download_policy
       end
 
       def associate_lifecycle_environments
-        self.lifecycle_environments = Katello::KTEnvironment.all if self.default_capsule?
+        self.lifecycle_environments = Katello::KTEnvironment.all if self.pulp_master?
+      end
+
+      #deprecated methods
```

Comment:
More opportunities for `alias`. I like the comment :+1: I'd also prefer the aliases be right next to their aliased method for better readability.

Reply:
now these i can alias :)